statsmodels/iolib/tests/test_pickle.py

```diff
@@ -42,3 +42,34 @@ def test_pickle():
     d = load_pickle(fh)
     fh.close()
     assert_equal(a, d)
+
+
+def test_pickle_supports_open():
+    tmpdir = tempfile.mkdtemp(prefix="pickle")
+    a = lrange(10)
+
+    class SubPath:
+        def __init__(self, path):
+            self._path = pathlib.Path(path)
+
+        def open(
+            self,
+            mode="r",
+            buffering=-1,
+            encoding=None,
+            errors=None,
+            newline=None,
+        ):
+            return self._path.open(
+                mode=mode,
+                buffering=buffering,
+                encoding=encoding,
+                errors=errors,
+                newline=newline,
+            )
```

Comment:
this looks really ugly, too much vertical space

Reply:
Embrace the 

![image](https://user-images.githubusercontent.com/5585221/125823897-6bf944dd-e23b-4b46-b32d-2e2442b06c79.png)

Welcoming black into your code frees you from even looking at formatting.

Reply:
On a serious note, the vertical space has the long-run effect of minimizing diffs.